README.md

```diff
@@ -207,7 +207,7 @@ invokes cmake commands as needed.
 * Change to the root of the source code directory, change to the most recent release branch, and build:
 
         cd monero
-        git checkout release-v0.14
+        git checkout release-v0.14.0.2
```

Comment:
This one does not exist either :P

Reply:
release-v0.14 does not exist, and you are right, I tried to correct it incorrectly! Wrong prefix, I will update that

Reply:
@moneromooo-monero ok, fixed! My humble suggestion, as a Git consultant in a former life, is to document soon-to-be-created tags in dev branches, and wait until those tags actually exist to merge those changes into master branch. That will prevent newbies like me from getting confused and retains the flexibility that devs want.